tests/ctags/kotlin.kt.tags

```diff
@@ -0,0 +1,12 @@
+# format=tagmanager
+AbstractWorker�32�com.example.helloworld�0
+Global�1�com.example.helloworld�0
+StringWorker�4096�com.example.helloworld�0
+Worker�1�com.example.helloworld�0
+com.example.helloworld�256�0
+greeting�16384�com.example.helloworld.Global�0
+main�16�com.example.helloworld�0
+name�16384�com.example.helloworld.Worker�0
+process�16�com.example.helloworld.AbstractWorker�0
+process�16�com.example.helloworld.Worker�0
+result�16384�com.example.helloworld.main�0
```

Comment:
Haven't checked but does the unit test cover all kinds from `TMParserMapEntry map_KOTLIN`?

Reply:
Yes, it covers all 8 tag kinds.